src/main/java/com/asu/cse545/group12/controller/UserController.java

```diff
@@ -63,13 +77,30 @@ public ModelAndView getForm() {
 		}
 		ModelAndView modelView = new ModelAndView();
 		modelView.addObject("user", new Users());
-		modelView.addObject("userPII", new UserPII());
+		modelView.addObject("userpii", new UserPII());
 		modelView.setViewName("signup");
 		
 		return modelView;
 	}
 
-	@RequestMapping("signUpExternalUser")
+	@RequestMapping( value= "signUpExternalUser")
+
+	public ModelAndView registerUser(@ModelAttribute Users user) {
+		//Users user=(Users)modelMap.get("user");
+		user.setUserStatus("A");
```

Comment:
Does A indicates active? we might want to insert record for signup as inactive once approved we can set it to active user.